lib/guardian/token/jwt.ex

```diff
@@ -119,7 +122,7 @@ defmodule Guardian.Token.Jwt do
   This will allow you to use the header values to determine dynamically the
   key that should be used.
 
-  ```
+  ```elixir
```

Comment:
@x-ji thank you very much for the help.

Before we merge it, can we refactor the block of code markdown syntax from this page?

The block of codes should be 4 spaces of indentation without the triple backticks since this syntax is not actually supported by `ex_doc`.

Reply:
I think ExDoc actually supports it now. See https://github.com/elixir-lang/elixir/pull/5389

And there was a fenced code block in this same file just a few lines above, which seems to work fine.

Though of course I can also change it to be indented.

Reply:
In fact it seems to me that ```elixir is being used all over the project. You can just run a search over the project. So if I want to refactor this then we'll have to refactor the whole project. Unless I misunderstood something.